microcosm_eventsource/tests/test_crud.py

```diff
@@ -102,16 +104,21 @@ def iter_events(self):
         yield reassigned
 
     def test_created_event(self):
-        response = self.client.post(
-            "/api/v1/task_event",
-            data=dumps(dict(
-                taskId=str(self.task.id),
-                eventType=TaskEventType.CREATED.name,
-            )),
-        )
+        created_event_id = new_object_id()
+        with patch.object(self.graph.task_event_store, "new_object_id") as mocked:
+            mocked.return_value = created_event_id
+            response = self.client.post(
+                "/api/v1/task_event",
+                data=dumps(dict(
+                    taskId=str(self.task.id),
+                    eventType=TaskEventType.CREATED.name,
+                    parentId=str(self.task.id),
+                )),
```

Comment:
This already delegated to the `store`'s new_object_id function, I just added this to exercise that logic.